src/client/pages/Embark/index.tsx

```diff
@@ -251,6 +251,20 @@ export const EmbarkRoot: React.FunctionComponent<EmbarkRootProps> = (props) => {
     }
   }, [props.name])
 
+  const isDenmarkInProduction =
```

Comment:
Was a bit confused by the name at first, but can't come up with a better one myself 😆